src/java.base/share/classes/java/lang/VersionProps.java.template

```diff
@@ -109,6 +109,10 @@ class VersionProps {
         props.put("java.class.version", CLASSFILE_MAJOR_MINOR);
 
         props.put("java.specification.version", VERSION_SPECIFICATION);
+
+        // Uncomment next props.put call after the first maintenance release for a
```

Comment:
Is "after" correct?  I think this value should be set to the target MR at the beginning of RI development for the release, similar to what is done for other version identification system properties.

Reply:
Well, sure -- this versioning info should be updated in the same way we update versions for all the early access builds of a release.